bocadillo_cli/main.py

```diff
@@ -48,7 +48,7 @@ def writefile(self, path: pathlib.Path, content: str):
             return
 
         if not self.dry:
-            with open(str(path), "w") as f:
+            with open(str(path), "w", encoding="utf-8") as f:
```

Comment:
Does this work on your end? I'm asking because if I remember well the SO answer mentioned using `io.open` instead of the `open` builtin.

Reply:
It worked, the SO comment is "Using io gives you backward compatibility with Python 2." so not required?

Reply:
Yup, sounds good to me.